source/includes/_endpoint-account.md

```diff
@@ -47,11 +47,114 @@ GET /account/
     "teams": "http://app.crunch.io/api/account/teams/",
     "projects": "http://app.crunch.io/api/account/projects/",
     "users": "http://app.crunch.io/api/account/users/",
-    "datasets": "http://app.crunch.io/api/account/datasets/"
+    "datasets": "http://app.crunch.io/api/account/datasets/",
+    "applications": "http://app.crunch.io/api/account/applications/"
   }
 }
 ```
 
+##### Applications
+
+```http
+GET /account/applications/
+```
+
+GET returns a Shoji Catalog with the list of all the configured subdomains an account has.
+
+```json
+{
+    "element":"shoji:catalog",
+    "index": {
+        "./mycompany/": {}
+    }
+}
+```
+
+POST a Shoji Entity here to make a new application. The `subdomain` must be
+unique system-wide and must be a valid URL segment; if unavailable or invalid, the server will return a 400 response.
```

Comment:
We allow letters, numbers, - according to the spec and a bit of cargo culture (cannot start or end on - and. following python recommendations, no numbers at the beginning). We always force lowercase.

Here's the rfc that indicates numbers letters and -: https://tools.ietf.org/html/rfc952